### PR TITLE
Patch BM nodeset to update all packages during bootstrap phase

### PR DESCRIPTION
**Pull request by raukadah**
In baremetal EDPM job, we use EDPM image to create EDPM node. it has all the required packages pre-installed during image build process.

If an older version of package got included during the image build process. Then during edpm_deploy, bootstrap role is not going to update the package to latest.

It is not advisable to use latest package installation from edpm-ansible bootstrap ansible role.

For CI case, we use edpm_bootstrap_command to update all the packages.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running

### CODE REVIEW

**Comment by raukadah** [comment]
tested here: https://review.rdoproject.org/r/c/testproject/+/51049/1#message-e7e8012f7e79f7089ca47603f1b74b3c8bcbb528

File: ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml
Comment (SeanMooney) [inline ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml:212]: this might mask actrul reall issue with edpm-ansible.
so I'm not sure its correct to do this in the ci-framework

i think adding a dnf update to the bootstrap role in edpm-ansibele is actually the correct fix.

**Review by SeanMooney — CHANGES_REQUESTED** [review]

File: ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml
Comment (raukadah) [reply ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml:212]: thank you @SeanMooney , Sounds good. I will update the same in edpm-ansible.

Comment (raukadah) [reply ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml:212]: We will merge this one to unblock integration line and then we will revert back once edpm-ansible pr is ready.

**Comment by cjeanner** [comment]
/approve

don't forget to revert once the proper fix in edpm-ansible is done.

File: ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml
Comment (viroel) [reply ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml:212]: @raukadah please revert this one after you get edpm-ansible pr merged. Thanks

**Comment by viroel** [comment]
/lgtm

Considering that the real fix will land at edpm-ansible and this can be reverted afterwards

**Pull request merged**
